src/main.rs

```diff
@@ -56,12 +56,7 @@ enum Commands {
         executor: bool,
     },
     #[command(about = "wipes the node and farm instance (along with your plots)")]
-    Wipe {
-        #[arg(long, action)]
-        farmer: bool,
-        #[arg(long, action)]
-        node: bool,
```

Comment:
I think you shouldn't remove flags. I think the path should be as follow:
```
if terminal && !farmer && !node {
  interactive_prompts();
} else {
  current_code();
}
```

Reply:
will think about that, thanks!

Reply:
`terminal` bool here should be check whether stdout is terminal. This crate will do the job I think: https://crates.io/crates/is-terminal